portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/SPA.testcase

```diff
@@ -33,4 +33,86 @@ definition {
 		AssertVisible.assertVisibleNoSPAWait(locator1 = "//html[contains(@class,'lfr-spa-loading')]");
 	}
 
+	@description = "Validate user can configure SPA timeout notification"
+	@priority = "4"
+	test ConfigureSPANotificationTimeout {
+		property test.name.skip.portal.instance = "SPA#AssertLoadingBar";
+
+		var appName = "Liferay Frontend JS SPA Web";
+
+		task ("Configure User Notification Timeout to 1 second") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "System Settings");
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "Infrastructure",
+				configurationName = "Frontend SPA Infrastructure",
+				configurationScope = "System Scope");
+
+			Type(
+				key_text = "User Notification Timeout",
+				locator1 = "TextInput#ANY",
+				value1 = "1000");
+
+			SystemSettings.saveConfiguration();
+		}
+
+		task ("Reset the SPA module in App Manager to fully apply the SPA configuration") {
+			Navigator.openURL();
+
+			ApplicationsMenu.gotoPortlet(
+				category = "System",
+				panel = "Control Panel",
+				portlet = "App Manager");
+
+			AppManager.viewAppCP(
+				appName = "${appName}",
+				appStatus = "Active",
+				searchTerm = "SPA");
+
+			AppManager.deactivateAppCP(appName = "${appName}");
+
+			AppManager.viewAppCP(
+				appName = "${appName}",
+				appStatus = "Resolved",
+				searchTerm = "SPA");
+
+			AppManager.activateAppCP(appName = "${appName}");
```

Comment:
This forces a full refresh which is what makes the setting take effect again, so you should be able to just skip all this task and refresh manually.

Reply:
will replace with refresh.